src/agent/command_store/include/command.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <limits>
+#include <string>
+
+namespace command_store
+{
+    enum class Status
+    {
+        SUCCESS,
+        ERROR,
+        INPROCESS,
+        TIMEOUT
+    };
+
+    class Command
+    {
+    public:
+        Command()
+            : m_id(std::numeric_limits<int>::min())
+            , m_status(Status::SUCCESS)
+            , m_time(0.0)
+        {
+        }
+
+        Command(
+            int id, const std::string& module, const std::string& command, const std::string& parameters, Status status)
+            : m_id(id)
+            , m_module(module)
+            , m_command(command)
+            , m_parameters(parameters)
+            , m_status(status)
+        {
+        }
+
+        int m_id {-1};
+        std::string m_module;
+        std::string m_command;
+        std::string m_parameters;
+        Status m_status;
```

Comment:
I think we also need an `m_result` and a function to update it and the status.

Reply:
Done